### PR TITLE
Updated StopGenMatch and added TrainingNTupleVars for stop reconstruction NN

### PR DESCRIPTION
**Pull request by amercald**

### CODE REVIEW

File: Framework/include/CommonVariables.h [hunk 1]
```diff
@@ -84,13 +84,10 @@ class CommonVariables
...
-        bool passHEMVetoLoose = !(objectInHEM(Muons,     -3.00, -1.30, -1.57, -0.87, 20.0, runYear) ||
-                                  objectInHEM(Electrons, -3.00, -1.30, -1.57, -0.87, 20.0, runYear));
-        tr.registerDerivedVar("passHEMVetoLoose"+myVarSuffix_, passHEMVetoLoose);
-
-        // Define full HEM15/16 veto
-        bool passHEMVeto = !objectInHEM(Jets,      -3.20, -1.10, -1.77, -0.67, 20.0, runYear) && passHEMVetoLoose; 
+        // Define HEM15/16 veto
+        bool passHEMVeto = !(objectInHEM(Jets,      -3.20, -1.10, -1.77, -0.67, 20.0, runYear) ||
+                             objectInHEM(Muons,     -3.00, -1.30, -1.57, -0.87, 20.0, runYear) || 
+                             objectInHEM(Electrons, -3.00, -1.30, -1.57, -0.87, 20.0, runYear));
```
Comment (cmadrid1) [inline Framework/include/CommonVariables.h:90]: Why did you remove the passHEMVetoLoose. Were you the only person using it?

File: Framework/include/TrainingNTupleVars.h [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        const auto& GoodJets_pt30           = tr.getVec<bool>("GoodJets_pt30"+myVarSuffix_);
+        const auto& GoodLeptons             = tr.getVec<std::pair<std::string, TLorentzVector>>("GoodLeptons"+myVarSuffix_);
+
+        
+
+
+
+
+
+
```
Comment (cmadrid1) [inline Framework/include/TrainingNTupleVars.h:22]: Lots of spaces it would be cleaner to remove them. Normally I wouldn't care but I added a comment for another file anyway.

**Comment by cmadrid1** [comment]
update me please

File: Framework/include/CommonVariables.h (hunk 1 above)
Comment (amercald) [reply Framework/include/CommonVariables.h]: Whoops not sure how that happened, I reset the file to the master version and committed it
